docs/examples/multiple_stands.md

```diff
@@ -64,13 +64,91 @@ hardpy-panel -dbp 5985 -wp 8001 tests
 
 ### storing reports
 
+We can use the database to store a list of reports. This will require a separate database.
+
 We can use the **reports** database, as in the example [CouchDB Load](couchdb_load.md).
-A separate database is required to store the list of reports.
-By default, reports are saved to the default database on port 5984, but you can assign another port.
-Configure database on port 5986 for storing reports in `conftest.py` in the test folder:
 
-```python
-loader = CouchdbLoader(CouchdbConfig(port=5986))
+
+#### pytest.ini
```

Comment:
Duplicate information about the `.ini` file in the `###projects` chapter and here.

Reply:
Deleted.